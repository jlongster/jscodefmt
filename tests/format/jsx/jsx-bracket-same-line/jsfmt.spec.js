run_spec(__dirname, ["flow", "typescript"], { jsxBracketSameLine: true });
