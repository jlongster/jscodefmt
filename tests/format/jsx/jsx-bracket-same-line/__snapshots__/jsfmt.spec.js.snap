// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`last_line.js - {"jsxBracketSameLine":true} format 1`] = `
====================================options=====================================
jsxBracketSameLine: true
parsers: ["flow", "typescript"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  ignoreStuff={true}
>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>

=====================================output=====================================
<SomeHighlyConfiguredComponent
  onEnter={this.onEnter}
  onLeave={this.onLeave}
  onChange={this.onChange}
  initialValue={this.state.initialValue}
  ignoreStuff={true}>
  <div>and the children go here</div>
  <div>and here too</div>
</SomeHighlyConfiguredComponent>;

================================================================================
`;
